### DISCUSSION TITLE
Commit problem - Github says that my files are large by mistake

### DISCUSSION THREAD

**Discussion by DjordjeOstojic** [community/community#23025] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all,</p>
<p>This is my first post here, so I hope that I am in the right place. I have searched for the solution for this problem, but without success.</p>
<p>So, below is the image of the HTML website that I have built on my HDD:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/f07c70f517af98832689154f243584cb7a85d7e4.png" data-download-href="/uploads/short-url/yjrcPQ6P73w6uAX5MvP3FMEqAN6.png?dl=1" title="1.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/25553919/181074256-34307fc6-b6cf-415d-913a-1df0ab00382a.png" alt="1.PNG" data-base62-sha1="yjrcPQ6P73w6uAX5MvP3FMEqAN6" width="533" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/f07c70f517af98832689154f243584cb7a85d7e4_2_533x500.png, https://aws1.discourse-cdn.com/github/original/1X/f07c70f517af98832689154f243584cb7a85d7e4.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/f07c70f517af98832689154f243584cb7a85d7e4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/f07c70f517af98832689154f243584cb7a85d7e4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">1.PNG</span><span class="informations">701×657 41.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>As you can see, there are 43 files, with total size of 6.33MB. When I try to upload these files to repository, I get the message that some files are too big, and only 14 files are recognized (picture below) </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d2f7af2d7bb2f8882830b02658aa2ea1865428f6.png" data-download-href="/uploads/short-url/u6iVEFbiowOzqxHI9sS6Il3Wp7w.png?dl=1" title="2.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/25553919/181074259-fdf80064-bdd5-4310-b2b5-a2abfea41d5e.png" alt="2.PNG" data-base62-sha1="u6iVEFbiowOzqxHI9sS6Il3Wp7w" width="690" height="275" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d2f7af2d7bb2f8882830b02658aa2ea1865428f6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2.PNG</span><span class="informations">976×389 8.07 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Does anybody know what is the problem?</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>The “using github” forum is probably more appropriate. But no problem.</p>
<p>You’re using drag-and-drop which I’ve found problematic in the past (I get similar behavior as you describe occasionally). Try dragging only a few of the files at a time, or at worst, one-by-one. You may have one file which is causing the problem and you may have to use a different mechanism to upload it.</p>
